### PR TITLE
Footer component

### PR DESCRIPTION
**Pull request by smolamarcin**
image

### CODE REVIEW

**Comment by smolamarcin** [comment]
Idk how move footbar to the bottom on the site permamently.

**Comment by michaljach** [comment]
You can use `position: fixed;` or `position: absolute;` or `flexbox`. Research and read about those to gain knowledge.

**Comment by smolamarcin** [comment]
Fixed position of footer. I used position: absolute.

**Comment by smolamarcin** [comment]
footer

**Comment by smolamarcin** [comment]
footer

File: src/app/footer/footer.component.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
+footer{
+  display: inline-block;
+}
+ul.social-network {
+  text-align: center;
+  list-style: none;
+	display: inline;
```
Comment (michaljach) [inline src/app/footer/footer.component.css]: Indentation

File: src/app/footer/footer.component.css [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+}
+.social-network a.icoGithub:hover {
+	background-color:#4078c0;
+}
+.social-circle li a {
+	display:inline-block;
+	position:relative;
+	margin:0 auto 0 auto;
+	-moz-border-radius:50%;
+	-webkit-border-radius:50%;
```
Comment (michaljach) [inline src/app/footer/footer.component.css]: Get rid off -moz.. prefixes, leave only `border-radius:50%`.

File: src/app/footer/footer.component.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+}
+.social-circle li i {
+	margin:0;
+	line-height:50px;
+	text-align: center;
+
+}
+
+.social-circle li a:hover i, .triggeredHover {
+	-moz-transform: rotate(360deg);
```
Comment (michaljach) [inline src/app/footer/footer.component.css]: Same here.

**Pull request merged**